src/menu/charprofile.c

```diff
@@ -0,0 +1,403 @@
+	},
+	[PROFILE_YOUMU] = {
+		.name = "youmu",
+		.fullname = "Konpaku Yōmu",
+		.title = "Swordswoman Between Worlds",
+		.description = "Species: Half-Human, Half-Phantom\nArea of Study: Fencing\n\nThe swordswoman of the somewhat-dead.\n\n“While you were playing games, I was studying the blade” … is probably what she’d say.\n\nEntirely too serious about everything, and utterly terrified of (half of) her own existence.\n\nStill, she’s been given an important mission by Lady Yuyuko. Hopefully she doesn’t let it get to her head.",
+		.background = "youmu_bombbg1",
+		.faces = FACES("normal", "eeeeh", "embarrassed", "eyes_closed", "chuuni", "happy", "relaxed", "sigh", "smug", "surprised", "unamused"),
+	},
+	[PROFILE_CIRNO] = {
+		.name = "cirno",
+		.fullname = "Cirno",
+		.title = "Thermodynamic Ice Fairy",
+		.description = "Species: Fairy\nField of Study: Thermodynamic Systems\nThe lovably-foolish ice fairy.\n\nPerhaps she’s a bit dissatisfied after her recent duels with secret gods and hellish fairies?\n\nConsider going easy on her.",
+		.background = "stage1/cirnobg",
+		.unlock = PBGM_STAGE1_BOSS,
+		.faces = FACES("normal", "angry", "defeated"),
+	},
+	[PROFILE_HINA] = {
+		.name = "hina",
+		.fullname = "Kagiyama Hina",
+		.title = "Gyroscopic Pestilence God",
+		.description = "Species: Pestilence God\nField of Study: Gyroscopic Stabilization\n\nGuardian of Yōkai Mountain. Her angular momentum is out of this world!\n\nShe seems awfully concerned with your health and safety, to the point of being quite overbearing.\n\nYou’re old enough to decide your own path in life, though.",
+		.unlock = PBGM_STAGE2_BOSS,
+		.background = "stage2/spellbg2",
+		.faces = FACES("normal", "concerned", "serious", "defeated"),
+	},
+	[PROFILE_SCUTTLE] = {
+		.name = "scuttle",
+		.fullname = "Scuttle",
+		.title = "???",
+		.description = "????",
+		.unlock = PBGM_STAGE3_BOSS,
+		.background = "stage3/spellbg1",
+		.faces = FACES("normal"),
+	},
+	[PROFILE_WRIGGLE] = {
+		.name = "wriggle",
+		.fullname = "Wriggle Nightbug",
+		.title = "Insect Rights Activist",
+		.description = "Species: Insect\nField of Study: Evolutionary Socio-Entomology\n\nA bright bug - or was it insect? - far from her usual stomping grounds.\nShe feels that Insects have lost their “rightful place” in history.\nIf she thinks she has a raw evolutionary deal, someone ought to tell her about the trilobites…",
+		.unlock = PBGM_STAGE3_BOSS,
+		.background = "stage3/spellbg2",
+		.faces = FACES("normal", "calm", "outraged", "proud", "defeated"),
+	},
+	[PROFILE_KURUMI] = {
+		.name = "kurumi",
+		.fullname = "Kurumi",
+		.title = "High-Society Phlebotomist",
+		.description = "Species: Vampire\nField of Study: Phlebotomy\n\nVampiric blast from the past. Doesn’t she know Gensōkyō already has new bloodsuckers in town?\n\nSucking blood is what she’s good at, but her current job is a gatekeeper, and her true passion is high fashion.\n\nEveryone’s got multiple side-hustles these days…",
```

Comment:
To avoid repeating phlebotomist, this could be hematology, but maybe repeating is good/ok.